src/ankit/Scanner/MaximumFrequencyWord.java

```diff
@@ -0,0 +1,37 @@
+/*Assignment-18 : max repeating word with its frequency.
+*/
+package ankit.Scanner;
+
+import java.util.Scanner;
+
+public class MaximumFrequencyWord {
+
+		void maxWordFrequency(String str) {
+			String[] strArray = new String[str.length()];
+			strArray = str.split(" ");
+			int maximumCount=0;
+			String word = null;
+			for(int index=0;index<strArray.length;index++) {
+				maximumCount = 0;
+				word = strArray[index];
+				for(int occurenceIndex=0;occurenceIndex<strArray.length;occurenceIndex++) {
```

Comment:
this occurenceIndex should start from index + 1, else if start it from 0 mean you are comparting string with self once during iteration.